code/modules/modular_computers/file_system/programs/arcade.dm

```diff
@@ -0,0 +1,165 @@
+/datum/computer_file/program/arcade
+	filename = "arcade"
+	filedesc = "Nanotransen Micro Arcade"
```

Comment:
Nano**transen** 👀

Reply:
Nanotrasen is most popular spelling, but I have heard it a few ways

Reply:
It's not an ArcaneMusic PR without at LEAST one misspelling.

Reply:
Feature